catroid/src/main/java/org/catrobat/catroid/utils/TrackingUtil.java

```diff
@@ -61,7 +61,7 @@ private TrackingUtil() {
 	}
 
 	public static void trackCreateProgram(String projectName, Boolean landscapeMode, boolean exampleProgram) {
```

Comment:
An even better approach might be to use a protection proxy.
I could implement a TrackingInvocationHandler that proxies the TrackingUtil class.
By doing that, all that needs to be done is to check:
if (BuildConfig.DATA_TRACKING) {
	return method.invoke(trackingUtil, method, args);
}
once in the invocation handlers invoke-method. So I don't even need a "blackHole"-Tracker.